Raygun.Druid4Net.IntegrationTests/Queries/TestQueryBase.cs

```diff
@@ -1,23 +1,29 @@
 ﻿using System;
-using System.Configuration;
+using System.IO;
+using Microsoft.Extensions.Configuration;
 
-namespace Raygun.Druid4Net.IntegrationTests.Queries
+namespace Raygun.Druid4Net.IntegrationTests.Queries;
+
+public abstract class TestQueryBase
 {
-  public abstract class TestQueryBase
-  {
-    protected DateTime FromDate = new DateTime(2016, 6, 27, 0, 0, 0, DateTimeKind.Utc);
-    protected DateTime ToDate = new DateTime(2016, 6, 28, 0, 0, 0, DateTimeKind.Utc);
+    private static readonly IConfigurationRoot Configuration = new ConfigurationBuilder()
```

Comment:
The indentation has changed to 4 spaces. It should be 2.
It might be worth adding in a `.editorconfig` file to set the indentation